src/vuestic-theme/vuestic-components/va-rating/VaRatingItem.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div
+    class="va-rating-item"
+    @keypress="onClickTab()"
+  >
+    <i
+      :class="iconClassesComputed"
+      class="va-rating-item__icon"
+      @mousemove="onHover"
+      @mouseleave="removeHover"
+      @click="onClick()"
+    />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'va-rating-item',
+  props: {
+    value: {
```

Comment:
I wonder why would you pass down value as `value: Boolean` + `halfIcon: Boolean` and emit value as `value: [0.5, 1]` instead of keeping it consistent.